packages/components/src/styles/hidden-links/hidden-links.stories.ts

```diff
@@ -40,16 +39,87 @@ export const Default = {
   render: (args: any) => {
     return generateTemplate({
       options: {
-        templateContent: html`<div class="relative bg-white w-full h-[256px] p-8">
+        templateContent: html`<div class="relative bg-white w-full h-[256px] p-8 flex">
           <div class="%CLASSES%">%SLOT%</div>
-          <p>Tab into this area to show a single button.</p>
+          <p class="self-center">Tab into this area to show a single button.</p>
         </div>`
       },
       args
     });
   }
 };
 
+/**
+ * Stack multiple `sd-hidden-links` by adding multiple of them to the same parent.
+ *
+ * **Hint:**
+ * Recommended for 2 or more links.
+ */
+export const StackLinks = {
+  render: () =>
+    html`<div class="relative bg-white w-full h-[256px] p-8 flex">
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Search</sd-button></div>
+      <div class="sd-hidden-links"><sd-button href="#">Skip to Content</sd-button></div>
+      <p class="self-center">Tab through this area to see buttons one after another.</p>
+    </div>`
+};
+
+/**
+ * Use the `&--multiple` class to show mutiple `sd-navigation-item` elements.
+ *
+ * **Hint:**
+ * Recommended for 2 or more links.
+ */
+export const MultipleLinks = {
```

Comment:
Multiple Links samples should say

Hidden Link 1
Hidden Link 2
Hidden Link 3

Reply:
Updated